azure-workload-identity-webhook.yaml

```diff
@@ -0,0 +1,56 @@
+package:
+  name: azure-workload-identity-webhook
+  version: 1.5.0
+  epoch: 1
+  description: Mutating webhook for Azure Workload Identity
+  copyright:
+    - license: MIT
+
+environment:
+  environment:
+    CGO_ENABLED: "0"
+    GOFLAGS: "-trimpath"
```

Comment:
we should consider omitting this, defaults of go/build pipeline is good. We want to use dynamic linking and avoid trimming the path.